automated install/basic-install.sh

```diff
@@ -2300,6 +2300,39 @@ copy_to_install_log() {
     chown pihole:pihole "${installLogLoc}"
 }
 
+disableLighttpd() {
+    # Detect if the terminal is interactive
+    if [[ -t 0 ]]; then
+        # The terminal is interactive
+        dialog --no-shadow --keep-tite \
+            --title "Pi-hole v6.0 does no longer need lighttpd" \
```

Comment:
```suggestion
            --title "Pi-hole v6.0 no longer uses lighttpd" \
```

Just to un-german the grammar